migrations/20160912132241_create_risks_of_bias.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+exports.up = (knex) => {
+  const schema = knex.schema;
+
+  schema.createTable('risk_of_biases', (table) => {
+    table.uuid('id').primary();
+
+    table.uuid('trial_id')
+      .notNullable()
+      .references('trials.id');
+    table.text('source_id')
+      .notNullable()
+      .references('sources.id');
+    table.text('source_url')
+      .notNullable();
+    table.text('study_id')
+      .notNullable();
+    table.timestamps(true, true);
+  });
+
+  schema.createTable('risk_of_bias_criterias', (table) => {
+    table.uuid('id').primary();
+
+    table.text('name')
+      .notNullable();
+    table.timestamps(true, true);
+  });
+
+  schema.createTable('risk_of_biases_risk_of_bias_criterias', (table) => {
+    table.uuid('risk_of_bias_id')
+    .references('risk_of_biases.id');
+    table.uuid('risk_of_bias_criteria_id')
+    .references('risk_of_bias_criterias.id');
+
+    table.enu('value', [
+      'yes',
+      'no',
+      'unknown',
+    ]).notNullable();
```

Comment:
I'm not sure where our previous chat about it went. If these are the only 3 valid states, and there's no possibility for a `null`, I'd prefer having it as a nullable boolean instead.